README.md

```diff
@@ -10,52 +10,54 @@ When you register a new account, you link the account to a community, using a pa
+**Automatic Installation**
 
-- Install Python. Konsent is written in Python 2.7, which can be installed from [the official Python website](https://www.python.org/ftp/python/2.7.14/Python-2.7.14.tar.xz).
+- Install Python 3.
 
-- Install the required Python packages. The easiest way to install Python packages is with Pip, which comes with Python 2.7 by default. The packages required to run Konsent are: `flask, flask-mysqldb, wtforms, passlib, functools, datetime`. To install with pip: `pip install datetime flask flask-mysqldb wtforms passlib functools`. Some of these packages may already be installed, if so, just skip those.
+- Clone Konsent using git, `git clone https://github,cm/dellitsni/Konsent/` or download and unzip somewhere safe.
 
-When everything is ready to go, open up a command line (terminal, konsole, command prompt or something along those lines - depends on your OS), `git clone` this repository, `cd` to the repository and start `flask_app.py` using Python: `python flask_app.py`. If you want to adjust some settings (such as the resting time for a post in phase 2 and 3), you can do so on one of the first lines in `flask_app.py`.
+- Install dependencies by running `python setup.py install`.
 
-## How to contribute
+- Start your MySQL/MariaDB server and create a new database called `konsent`.
 
-You don't have to know anything about programming or the likes to contribute to Konsent - developing the concept further is currently just as important. In the future, discussion and concept development will probably happen on the subraddle /f/konsent, on Raddle.me. Until then we can use GitHub's issues.
+- Populate the database by running `python konsent createdb`.
 
-Contributing to the code-base is easy if you know Python and the basics of Git, and there's lots to be done. Simply install all the dependencies, as explained in the chapter above, grab your favorite code editor, take a look at the open issues and fire away. We're following the branching model explained [here](https://nvie.com/posts/a-successful-git-branching-model/) loosely. We're following the [PEP8](https://pep8.org/) style guide.
+**Manual Installation**
 
-You can ship your changes pretty much however you want, although a good old pull-request is preferred.
+- Install Python 3.
 
-## Changelog
+- Clone Konsent using git, `git clone https://github.com/dellitsni/Konsent/` or download and unzip somewhere safe.
 
-### v. 2.0a
+- Use `pip` to install the dependencies: `pip install flask flask-mysqldb flaks-sqlalchemy wtforms bcrypt`. Should be executed as superuser if you're on Linux.
 
-- Got rid of "display name" and made some rules for usernames
+- Install `konsent` as a module in develop mode by running `pip`: `pip install konsent -e` in Konsent's parent folder.
```

Comment:
this is only if you actually want develop mode for development, otherwise you just install it the regular way